src/app/api/postTerms.ts

```diff
@@ -0,0 +1,28 @@
+import { apiWrapper } from "./apiWrapper";
+
+interface TermsPayload {
+  address: string;
+  termsAccepted: boolean;
+  publicKey: string;
+}
+
+export const postTerms = async (
+  address: string,
+  termsAccepted: boolean,
+  publicKey: string,
+) => {
```

Comment:
use an object as a single param to increase readability

Reply:
We should also denote a return type. It looks like the return type is boolean on whether the request is successful or not. This should be denoted in a comment as well.